apis/management.cattle.io/v3/schema/schema.go

```diff
@@ -506,7 +506,25 @@ func authnTypes(schemas *types.Schemas) *types.Schemas {
 		MustImportAndCustomize(&Version, v3.KeyCloakConfig{}, configSchema).
 		MustImportAndCustomize(&Version, v3.OKTAConfig{}, configSchema).
 		MustImport(&Version, v3.SamlConfigTestInput{}).
-		MustImport(&Version, v3.SamlConfigTestOutput{})
+		MustImport(&Version, v3.SamlConfigTestOutput{}).
+		//GoogleOAuth Config
+		MustImportAndCustomize(&Version, v3.GoogleOAuthConfig{}, func(schema *types.Schema) {
+			schema.BaseType = "authConfig"
+			schema.ResourceActions = map[string]types.Action{
+				"disable": {},
+				"configureTest": {
+					Input:  "googleOAuthConfig",
```

Comment:
Lowercase `A` (`googleOauthConfig`) throughout.  At least in what's visible to the client, if go linting wants it capital in the Go types that's fine